fintech-examples/fintech-ui/src/app/common/share.module.ts

```diff
@@ -1,13 +1,15 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
-import { ReactiveFormsModule } from '@angular/forms';
-import { HttpClientModule } from '@angular/common/http';
-import { InfoModule } from '../errorsHandler/info/info.module';
-import { SearchComponent } from './search/search.component';
+import { NgModule } from "@angular/core";
```

Comment:
No double quotes, looks like you tslint does not work. I have same problem.

Reply:
And I dont understand why double quotes should not be used. Nearly all ts files in node_modules use double quotes. It's Collins who complained about double quotes in fintech-ui. I am fine with double quotes.

Reply:
fixed